hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueHandler.java

```diff
@@ -378,7 +387,11 @@ ContainerCommandResponseProto handleCreateContainer(
     try {
       if (containerSet.getContainer(containerID) == null) {
         newContainer.create(volumeSet, volumeChoosingPolicy, clusterId);
-        created = containerSet.addContainer(newContainer);
+        if (RECOVERING == newContainer.getContainerState()) {
+          created = containerSet.addContainerByOverwriteMissingContainer(newContainer);
+        } else {
+          created = containerSet.addContainer(newContainer);
+        }
```

Comment:
```suggestion
        if (RECOVERING == newContainer.getContainerState()) {
          created = containerSet.addContainer(newContainer);
        } else {
          created = containerSet.addContainerIfNotPresent(newContainer);
        }
```

Reply:
If I rename the method name it would lead to change everywhere. Increasing the number of files again.